Exercices/JS05/index.js

```diff
@@ -0,0 +1,9 @@
+"use strict"; 
+alert("Enter two numbers to see if the first is greater than the second");
+let fristNumber = prompt("Insert frist number:");
+let secondNumber = prompt("Insert second number:");
+if (fristNumber > secondNumber) {
+    alert("Frist number is greater");
+} else {
+    alert("Second number is greater");
+}
```

Comment:
Your condition doesn't cover all possible cases. There's something missing 🤔  Think about it and improve it.